### PR TITLE
#183 create reusable input component

### PR DESCRIPTION
**Pull request by Jasminamih**

### CODE REVIEW

File: components/reusable-components/text-input/TextInput.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+  const formik = useFormik({
+    initialValues: { [name]: '' },
+    validationSchema: Yup.object({
+      name: Yup.string()
+        .required('Полето е задолжително')
+        .matches(
+          inputRegex,
+          '*Дозволени се следните карактери.... името може да биде на кирилица и латиница'
+        ),
```
Comment (edichoska) [inline components/reusable-components/text-input/TextInput.tsx:25]: This looks good so far.
This input will need to accept:

- validation schema
- placeholder
- error message?
- the input type can be: text, email, number
- name would be also dynamically added
-

Comment (edichoska) [reply components/reusable-components/text-input/TextInput.tsx:25]: Also the input should be able to accept props, We might need to use for example: disabled

**Comment by stoimenov95** [comment]
@Jasminamih can we also have an example of the input (implementation for the newsletter maybe) so we can see the design as well?

**Review by edichoska — APPROVED** [review]
👏

**Pull request merged**